app/index.html

```diff
@@ -15,6 +15,15 @@
     <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,600,300,700' rel='stylesheet' type='text/css'>
 
     {{content-for "head-footer"}}
+    <script type="text/javascript"
+        src="//cdnjs.cloudflare.com/ajax/libs/mathjax/2.6.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
+    </script>
+    <script type="text/x-mathjax-config">
```

Comment:
I notice that this configuration is done in `index.html`, and that actual pages just call `Typeset`.

In the AM, can we touch base on this? I've not used MathJax before and want to be sure I understand what this is doing. 

In particular, I want to be sure this doesn't impose a performance penalty if MathJAX might try to do something on every page view / render, even if most pages don't need it.